exercises/firmware_auditing/README.md

```diff
@@ -245,4 +237,7 @@ There are other pieces of information that the linker exposes, which we do not u
 
 [hardware access control]: ../hardware_access_control/README.md#part-3
 
-Other ideas might include writing a policy that combines the above exercises, or integrating it into the `xmake` build system so that a given policy is automatically run when building your firmware image.
+Another idea is to write a policy that combines the above exercises, or integrating it into the `xmake` build system so that a given policy is automatically run when building your firmware image.
+Finally you can have a look at the [audit example][] in the RTOS repository.
+
+[audit example]: https://github.com/CHERIoT-Platform/cheriot-rtos/tree/main/examples/10.audit
```

Comment:
Please always refrain from using GitHub file links for any file in the `cheriot-rtos` or `sonata-software` repositories.

Reply:
Our submodule of CHERIoT RTOS doesn't include this example yet. I'll just remove this link for now.